### DISCUSSION TITLE
Insert new page in the middle, rename sibling pages(if numerical ordering exists)

### DISCUSSION THREAD

**Discussion by sanjarcode** [zim-desktop-wiki/zim-desktop-wiki#2060] [Ideas]
**Enhancement Request**
I wanted to add a page in the middle of my notebook(pages are named alphabetically) To do this, I have to:
    1. Insert the page where I want.
    2. Rename all the siblings(ahead of it)
This can be a real pain.
![image](https://user-images.githubusercontent.com/48156230/84747476-ec439100-afd4-11ea-96ab-3fcad60810c4.png) ![image](https://user-images.githubusercontent.com/48156230/84747537-054c4200-afd5-11ea-9425-f8777f5a8cd5.png)

**Alternatives considered**
I wrote a script to do this, which renamed the text files and the folders, but it messed up backlinks, as Zim is not *aware* of the changes my script made.

It's a good plugin idea, I think.

**Comment by Nick222** [comment]
You want to preserve numbering while inserting an new page?

**Comment by sanjarcode** [comment]
Yes
